db/migrate/20210405192922_remove_gene_search_flag.rb

```diff
@@ -0,0 +1,22 @@
+class RemoveSpatialFeatureFlag < Mongoid::Migration
```

Comment:
```suggestion
class RemoveGeneStudyFilterFlag < Mongoid::Migration
```

Or another related name.